src/utils/setupHooks.js

```diff
@@ -1,12 +1,7 @@
-import reporter from './reporter';
-
 export default function setupHooks(context) {
   function invalid(callback) {
     if (context.state) {
-      reporter(context.options, {
-        log: context.log,
-        state: false,
-      });
+      context.options.reporter(context);
```

Comment:
@evilebottnawi I rebased from the latest next branch and then this code can't run. Could you fix this? Thanks.

```

    TypeError: context.options.reporter is not a function

      30 | 
      31 |       // print webpack output
    > 32 |       context.options.reporter(context, stats);
         |                       ^
      33 | 
      34 |       // execute callback that are delayed
      35 |       const { callbacks } = context;

      at process.nextTick (src/utils/setupHooks.js:32:23)
```